pandas/tests/reshape/merge/test_multi.py

```diff
@@ -0,0 +1,598 @@
+# pylint: disable=E1103
+
+from collections import OrderedDict
+
+import numpy as np
+import pytest
+from numpy import nan
+from numpy.random import randn
+
+import pandas as pd
+import pandas.util.testing as tm
+from pandas import (DataFrame, Index, MultiIndex, Series)
+from pandas.compat import lzip
+from pandas.core.reshape.concat import concat
+from pandas.core.reshape.merge import merge
+from pandas.util.testing import assert_frame_equal
+
+
+class TestMergeMulti(object):
+
+    def setup_method(self):
```

Comment:
in a followup (or here if it works), like to use fixtures

Reply:
These tests were not written by me but I can amend them. I fixtured the first set of tests

Reply:
can you create an issue to fixturize this